lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,36 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance, id: :db_instance_identifier
+
+  attribute :db_instance_identifier, kind_of: String, name_attribute: true
+
+  attribute :engine, kind_of: String
+  attribute :engine_version, kind_of: String
+  attribute :db_instance_class, kind_of: String
+  attribute :multi_az, default: false, kind_of: [TrueClass, FalseClass]
+  attribute :allocated_storage, kind_of: Integer, default: 5
```

Comment:
@tyler-ball Heya, was looking at these comments.  Gonna hold off on more changes until you commit.  I think @tylercloke has addressed the documentation needs.